apps/dashboard/src/components/side-navigation/side-navigation.tsx

```diff
@@ -153,6 +155,12 @@ export const SideNavigation = () => {
                 <RiKey2Line className="size-4" />
                 <span>API Keys</span>
               </NavigationLink>
+              {isWebhooksManagementEnabled && !IS_SELF_HOSTED && (
```

Comment:
i wonder if the IS_SELF_HOSTED is needed, the FF won't make the work by itself?
```suggestion
              {isWebhooksManagementEnabled (
```

Reply:
It was more for when we remove the FF, to not forget about it. But I added a CTA for cloud on self-hosted